packages/react-reconciler/src/ReactFiberCallUserSpace.js

```diff
@@ -177,11 +179,22 @@ export const callComponentWillUnmountInDEV: (
 
 const callCreate = {
   'react-stack-bottom-frame': function (effect: Effect): (() => void) | void {
-    const create = effect.create;
-    const inst = effect.inst;
-    const destroy = create();
-    inst.destroy = destroy;
-    return destroy;
+    if (enableUseResourceEffectHook && typeof effect.destroy === 'function') {
+      const inst = effect.inst;
```

Comment:
If I delete this block, no tests fail. Can we add a test for this?

Reply:
hmm this was a leftover that i don't need anymore, will clean it up